core/arch/arm/mm/core_mmu.c

```diff
@@ -338,6 +361,10 @@ static void init_mem_map(struct tee_mmap_region *memory_map, size_t num_elems)
 		}
 		add_phys_mem(memory_map, num_elems, &m, &last);
 	}
+
+	add_va_space(memory_map, num_elems, MEM_AREA_RES_VASPACE,
+		     RES_VASPACE_SIZE, &last);
```

Comment:
minor: `MEM_AREA_RES_VASPACE` does not need to be a parameter. maybe the paddr and byte size.